packages/libspatialindex.rb

```diff
@@ -19,5 +32,6 @@ def self.build
 
   def self.install
     system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
+    FileUtils.mv "#{CREW_DEST_PREFIX}/lib", CREW_DEST_LIB_PREFIX if ARCH == 'x86_64'
```

Comment:
That could be a simple way to fix it without patching:)

Reply:
Does this also have pkgconfig files though which have to be patched too?

Reply:
Not sure, what is the output of `crew files libspatialindex`?